hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx

```diff
@@ -104,11 +105,17 @@ const withTaskListContext = (WrappedComponent) => {
     };
 
     openTask = (index) => {
-      this.setState(prevState => ({ activeTask: prevState.tasks[index] }));
+      this.setState(prevState => ({
+        activeTask: prevState.tasks[index],
+        opened:     index,
```

Comment:
I think it's a bit confusing to use index or list item for extremely similar purposes. Let's use either index or full item in both cases.

Reply:
Same about naming: we have `chosen`, `opened` and `activeTask`. Either all with postfix `Task` or all without.
More say, don't you think that `opened` and `activeTask` are the same things?

Reply:
Yep, you're right. I thought about it but something seemed strange in my mind 😕